### PR TITLE
fix(accessibility): auth app tabbing and focusing fixes #ACC-15

### PR DESCRIPTION
**Pull request by BartDyb**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-15" title="ACC-15" target="_blank">SubtaskACC-15</a>  9.2.1.1 Fix auth app tabbing issues
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Fixes many issues related to correct tabbing and focusing clickable elements in the auth app.

https://wearezeta.atlassian.net/browse/ACC-15

### CODE REVIEW

File: src/script/auth/component/LinkButton.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
```
Comment (ffflorian) [inline src/script/auth/component/LinkButton.tsx]:
```suggestion
 * Copyright (C) 2022 Wire Swiss GmbH
```

Or did this file already exist?

File: src/script/auth/page/SetAccountType.tsx [hunk 2]
```diff
@@ -48,14 +46,24 @@ interface Props extends React.HTMLProps<HTMLDivElement> {}
...
-  const {history} = useReactRouter();
   const isMacOsWrapper = Runtime.isDesktopApp() && Runtime.isMacOS();
 
   const backArrow = (
     <RouterLink to={ROUTE.INDEX} data-uie-name="go-index">
       <ArrowIcon direction="left" color={COLOR.TEXT} style={{opacity: 0.56}} />
     </RouterLink>
   );
+
+  const iconStyles = {
```
Comment (ffflorian) [inline src/script/auth/page/SetAccountType.tsx]:
```suggestion
  const iconStyles: React.CSSProperties = {
```

**Review by ffflorian — APPROVED** [review]

File: src/script/auth/component/LinkButton.tsx (hunk 1 above)
Comment (BartDyb) [reply src/script/auth/component/LinkButton.tsx]: no, it's the new one - I'll fix it, thanks

File: src/script/auth/page/SetAccountType.tsx (hunk 2 above)
Comment (BartDyb) [reply src/script/auth/page/SetAccountType.tsx]: 👍

File: src/script/auth/component/AccountForm.tsx [hunk 3]
```diff
@@ -230,12 +233,13 @@ const AccountForm = ({account, ...props}: Props & ConnectedProps & DispatchProps
...
         markInvalid={!validInputs.terms}
         name="accept"
+        id="accept"
         required
         checked={registrationData.termsAccepted}
         data-uie-name="do-terms"
         style={{justifyContent: 'center'}}
       >
-        <CheckboxLabel>
+        <CheckboxLabel htmlFor="name">
```
Comment (atomrc) [inline src/script/auth/component/AccountForm.tsx]: Is it really for the `name` input? It feels weird, doesn't it?

Comment (BartDyb) [reply src/script/auth/component/AccountForm.tsx]: ach, you are right - it's for `accept` checkbox - I'll fix it

**Pull request merged**